### PR TITLE
Copter: Append the flight mode number to the message

### PR DESCRIPTION
**Pull request by muramura**
There is a time lag between the response to the MAVLINK message and the GCS message.
I can convert the GCS message to a flight name if there is a flight mode number in the message.

Screenshot from 2021-06-13 01-11-43

### CODE REVIEW

File: ArduCopter/mode.cpp [hunk 1]
```diff
@@ -199,7 +199,7 @@ bool Copter::set_mode(Mode::Number mode, ModeReason reason)
 
     Mode *new_flightmode = mode_from_mode_num((Mode::Number)mode);
     if (new_flightmode == nullptr) {
-        gcs().send_text(MAV_SEVERITY_WARNING,"No such mode");
+        gcs().send_text(MAV_SEVERITY_WARNING,"No such mode %d", uint8_t(mode));
```
Comment (peterbarker) [inline ArduCopter/mode.cpp:202]:
```suggestion
        gcs().send_text(MAV_SEVERITY_WARNING,"No such mode %u", uint8_t(mode));
```

**Review by peterbarker — CHANGES_REQUESTED** [review]
Equivalent change should be made in Blimp.  Let's try not to have *5* different versions of the same stuff.

For reference, Plane says
```
        gcs().send_text(MAV_SEVERITY_INFO, "Error: invalid mode number: %u", (unsigned)new_mode);
```

and Tracker/Sub/Rover say nothing.

We should be consistent across all our vehicles with this error message.

Plane's error wording is better but shouldn't have `Error:` in front of it if we're sending it as `MAV_SEVERITY_INFO` :-)

**Comment by muramura** [comment]
@peterbarker san.
In case of INFO at GCS message level, QGC does not give voice notification.
This is coded to, voice notification of GCS messages above NOTICE.
We want this PR message to be voice-delivered, so we will leave it as it is.

**Review by peterbarker — CHANGES_REQUESTED** [review]
Please move Plane's no-such-mode function up to AP_Vehicle and call that instead.

**Comment by muramura** [comment]
@peterbarker san. I have reflected on the review comments.

**Pull request closed without merging**